internal/spice/config_test.go

```diff
@@ -160,6 +162,20 @@ func TestIntegrationConfig_loadFromGit(t *testing.T) {
 				Level string `config:"level"`
 			}{Level: "hot"},
 		},
+		{
+			name: "Shorthands",
+			config: text.Dedent(`
+				[spice.shorthand]
+				can = commit amend --no-edit
+				wip = commit create -m \"wip: \\\"quoted\\\"\"
+				poorly = \"  # ignored (not a valid shorthand)
+			`),
```

Comment:
Is it needed to escape as you are using string literal ? (Backtick)


```suggestion
			config: text.Dedent(`
				[spice.shorthand]
				can = commit amend --no-edit
				wip = commit create -m "wip: \"quoted\""
				poorly = "  # ignored (not a valid shorthand)
			`),
```

Reply:
Ok